openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -926,14 +926,13 @@ If the Credential Response is not encrypted, the media type of the response MUST
 
 The following claims are used in the JSON-encoded Credential Response body:
 
-* `format`: REQUIRED. String denoting the format of the issued Credential.
 * `credential`: OPTIONAL. Contains issued Credential. MUST be present when `transaction_id` is not returned. MAY be a string or an object, depending on the Credential format. See (#format_profiles) for the Credential format specific encoding requirements.
 * `transaction_id`: OPTIONAL. String identifying a Deferred Issuance transaction. This claim is contained in the response if the Credential Issuer was unable to immediately issue the Credential. The value is subsequently used to obtain the respective Credential with the Deferred Credential Endpoint (see (#deferred-credential-issuance)). It MUST be present when the `credential` parameter is not returned. It MUST be invalidated after the Credential for which it was meant has been obtained by the Wallet.
 * `c_nonce`: OPTIONAL. String containing a nonce to be used to create a proof of possession of key material when requesting a Credential (see (#credential_request)). When received, the Wallet MUST use this nonce value for its subsequent Credential Requests until the Credential Issuer provides a fresh nonce.
 * `c_nonce_expires_in`: OPTIONAL. Number denoting the lifetime in seconds of the `c_nonce`.
 * `notification_id`: OPTIONAL. String identifying an issued Credential that the Wallet includes in the Notification Request as defined in (#notification ). This parameter MUST NOT be present if `credential` parameter is not present.
 
-The `format` key determines the Credential format and encoding of the credential in the Credential Response. Details are defined in the Credential Format Profiles in (#format_profiles). 
+The Credential format and encoding of the `credential` MUST match the Credential format specified in the Credential Request. Details are defined in the Credential Format Profiles in (#format_profiles).
```

Comment:
Encoding is not used anywhere else in this spec like this. I suggest to remove it.
```suggestion
The Credential format of the `credential` MUST match the Credential format specified in the Credential Request. Details are defined in the Credential Format Profiles in (#format_profiles).
```

Reply:
we need to put somewhere in the spec that "The `format` parameter value determines the Credential format and encoding of the credential in the Credential Response.". maybe it does not belong here anymore, but it should not be just omitted like this.

Reply:
```suggestion
The Credential format and encoding of the `credential` is determined by the Credential format specified in the Credential Request. Details are defined in the Credential Format Profiles in (#format_profiles).
```

Reply:
I commited @Sakurann comment as it does not affect @awoie proposal to remove encoding. I think we need more input for keeping or removing encoding

Reply:
I'm good with @Sakurann's new wording; it is accurate, the encoding of the credential does vary depending on the credential format (e.g. VCI requires that mdoc is base64url encoded in the response).

Reply:
IMO, it is an unnecessary repetition but I'm ok with either way. To me that sentence is just a bit confusing since encoding is always implied by the format. There is no format identified by its format identifier with multiple encodings. If we have such a case, we should fix that.

Reply:
I actually agree with Oliver and only kept encoding in there to keep the changes as small is possible for this late PR

Reply:
From the conversations I am having, it is not straightforward to the implementers that in the VP Token, sometimes credential is a string and sometimes it is an object. so we should be clear even if it feels redundant.

Reply:
It is similar to the EdDSA problem in JOSE where it could mean Ed443 or Ed25519. We should define format identifiers unambiguously.

Reply:
Also encoding is a characteristic of specific formats. Usually a format has one encoding. If there are legacy cases such as VCDM 1.1/1.0 where there are more encodings, we define already format identifiers like `jwt_vc_json`, and so on, that includes the encoding.